tests/e2e/support/objects/app-files/resource/index.ts

```diff
@@ -70,8 +70,9 @@ export class Resource {
     const startUrl = this.#page.url()
     await uploadResource({ ...args, page: this.#page })
     await this.#page.goto(startUrl)
-    // why? o_O
-    await this.#page.locator('body').click()
+    if (!config.ocis) {
+      await this.#page.locator('body').click()
```

Comment:
did you find out why the `body` click is needed in oc10 environment?

Reply:
IMO, it can be removed from here. But I have no idea why that was added before

Reply:
Yes. After changing the link role from uploder to editor  -> on the UI the role does not change (I couldn't reproduce it localy). I tried to set `page.reload()` but sometimes it fails. needs to be investigated and fixed in another PR
changing role - looks okay
![Screenshot 2023-04-18 at 08 28 28](https://user-images.githubusercontent.com/84779829/232690530-cf932d7c-43e3-4b58-9e20-e0e029efc9eb.png)

checking role - role does not change
![Screenshot 2023-04-18 at 08 29 11](https://user-images.githubusercontent.com/84779829/232690646-042579a5-ba31-4dcc-ae1a-410107eb3339.png)